Description: Fixes #873599 FTBFS explicit
 No further action needed, patch was taken with description from
 upstream and can disappear with the next release.
Last-Update: 2017-09-03

--- lxqt-panel-0.11.1.orig/plugin-kbindicator/src/x11/kbdlayout.cpp
+++ lxqt-panel-0.11.1/plugin-kbindicator/src/x11/kbdlayout.cpp
@@ -33,8 +33,15 @@
 
 #include <xkbcommon/xkbcommon-x11.h>
 #include <xcb/xcb.h>
+
+// Note: We need to override "explicit" as this is a C++ keyword. But it is
+// used as variable name in xkb.h. This is causing a failure in C++ compile
+// time.
+// Similar bug here: https://bugs.freedesktop.org/show_bug.cgi?id=74080
 #define explicit _explicit
 #include <xcb/xkb.h>
+#undef explicit
+
 #include "../kbdinfo.h"
 #include "../controls.h"
 
